style.css

```diff
@@ -652,31 +814,120 @@ span {
     grid-template-rows: repeat(18, 100vh);
   }
 
-  .general {
+  #p1 {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    width: 80%;
+    margin-left: 10%;
+    grid-column-gap: 20px;
+  }
+
+  #general_inf {
     grid-column: 2 / 1 span;
-    align-self: center;
   }
 
-  .project {
+  .project footer .external_link {
+    margin-left: 0;
+    width: 18%;
+    padding: 2% 0;
+  }
+
+  .skills {
+    display: flex;
+    align-items: baseline;
+    justify-content: flex-start;
+    padding-left: 0;
+    margin-left: 0;
+  }
+
+  footer .skills { margin-left: 0; }
+
+  #p1 footer .external_link {
+    margin-left: 0;
+    width: 18%;
+    padding: 2% 0;
+  }
+
+  #p1 footer .skills { margin-left: 0; }
+
+  #p2 {
     display: grid;
     grid-template-columns: 50% 50%;
+    margin-bottom: 100px;
     width: 80%;
-    height: 90%;
     margin-left: 10%;
     grid-column-gap: 20px;
+  }
+
+  #general_inf_1 {
+    grid-column: 2 / 1 span;
+  }
+
+  #p2 footer .external_link {
+    margin-left: 0;
+    width: 18%;
+    padding: 2% 0;
+  }
+
+  #p2 footer .skills { margin-left: 0; }
+
+  #p3 {
+    display: grid;
+    grid-template-columns: 50% 50%;
     margin-bottom: 100px;
+    width: 80%;
+    margin-left: 10%;
+    grid-column-gap: 20px;
   }
 
-  #general_inf {
+  #general_inf_2 {
     grid-column: 2 / 1 span;
   }
 
-  .project footer .external_link {
+  #p3 footer .external_link {
+    margin-left: 0;
+    width: 18%;
+    padding: 2% 0;
+  }
+
+  #p3 footer .skills { margin-left: 0; }
+
+  #p4 {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    margin-bottom: 100px;
+    width: 80%;
+    margin-left: 10%;
+    grid-column-gap: 20px;
+  }
+
+  #general_inf_3 {
+    grid-column: 2 / 1 span;
+  }
+
+  #p4 footer .external_link {
     margin-left: 0;
     width: 18%;
     padding: 2% 0;
   }
 
+  #p4 footer .skills { margin-left: 0; }
+
+  .general {
+    grid-column: 2 / 1 span;
+    align-self: center;
+  }
+
+  .project {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    width: 80%;
+    height: 90%;
+    margin-left: 10%;
+    grid-column-gap: 20px;
+    margin-bottom: 100px;
+  }
+
   .project_image {
```

Comment:
- [ ] You have done great to ensure that the project detail popup has a projected image. However, we should try as much as possible to ensure that we stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=Z3hCN1aRrx7pYjUK-0). Therefore because of this reason, I suggest that we ensure the image covers at least 95%+ width of the popup window. Implementing this, you should ensure the design matches the Figma design.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/225566681-9f5bebd6-04ae-4b44-a437-d162c89b05d7.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/225566780-5138a5bf-c717-483d-82e7-9dd58bae7ea0.png)

Reply:
Thank you for letting me know. the issue has been rectified.